features/pages/location_page.rb

```diff
@@ -0,0 +1,10 @@
+module Pages
+  class Location < SitePrism::Page
+    set_url '/locations/{id}'
```

Comment:
Is this a conscious decision to include the resource type in the URL?

We haven't followed that convention with guides for example.